File: jiggyjiggy/command/example/many/button/Light.java
package command.example.many.button;

public class Light {
	private final String spaceName;
	public Light(String spaceName) {
		this.spaceName = spaceName;
	}
	
	public void on() {
		System.out.println(this.spaceName + " " + Light.class.getSimpleName() + " - turn on");
	}
	
	public void off() {
		System.out.println(this.spaceName + " " + Light.class.getSimpleName() + " - turn off");
	}
}

File: yeongbinim/packages/headfirst/06_command/Command/Light.ts
class Light {
  name: string;
  constructor(name: string) {
    this.name = name;
  }
  on(): void {
    console.log(`${this.name} 전구를 켭니다.`);
  }
  off(): void {
    console.log(`${this.name} 전구를 끕니다.`);
  }
}

export default Light;
